format/Flight.proto

```diff
@@ -189,6 +191,31 @@ message Result {
   bytes body = 1;
 }
 
+/*
+ * The result of the CancelFlightInfo action.
+ *
+ * The result should be stored in Result.body.
+ */
+message ActionCancelFlightInfoResult {
+  enum CancelResult {
```

Comment:
We may want to remove `Cancel`:

```suggestion
  enum Result {
```

Or we may want to move this to top-level and return this enum directly instead of wrapping by `ActionCancelFlightInfoResult` message.

Reply:
At least in Protobuf/gRPC itself, you can only return messages and not bare enums